mosip_master/data_upgrade/1.1.5.5_to_1.2.0.1/migration-ui_spec.py

```
@@ -0,0 +1,814 @@
+ageGroupRequiresGuardian = []
+
+def getSupportedAgeGroups():
+	agegroup_config_json=json.loads(agegroup_config)
+	for ageGroup in agegroup_config_json.keys():
+		modalities = []
+		while not modalities:
+			modalities = agegroup_config_json.get(ageGroup).get("bioAttributes")
+		ageGroupBasedModalities[ageGroup] = modalities
+
+		requiresGuardianAuth = agegroup_config_json.get(ageGroup).get("isGuardianAuthRequired")
+		if(requiresGuardianAuth == True):
```

Ruff (0.15.15)

[error] Avoid equality comparisons to `True`; use `requiresGuardianAuth:` for truth checks

Replace with `requiresGuardianAuth`

(E712)